client.go

```diff
@@ -738,16 +766,15 @@ type LoginActionData = map[string]string
 
 type IdentitySessionIntermediateResponse = identityClient.IdentitySessionRequestResponse
 
+// TozIDLoginRequest is used to login to a TozID account to get a ToznySDKV3 or active TozID session (future plan)
 type TozIDLoginRequest struct {
-	Username  string
-	Password  string
-	RealmName string
-
-	APIBaseURL string
-	LoginHandler func(response *IdentitySessionIntermediateResponse) (LoginActionData, error)
```

Comment:
We should make sure that `func(response *IdentitySessionIntermediateResponse)` is the type for LoginHandler